draft-ietf-quic-ack-frequency.md

```diff
@@ -323,24 +323,59 @@ strategy.
 
 As specified in {{Section 13.2.1 of QUIC-TRANSPORT}}, endpoints are expected to
 send an acknowledgement immediately on receiving a reordered ack-eliciting
-packet. This extension modifies this behavior.
+packet. This extension modifies that behavior when an ACK_FREQUENCY frame with
+a Reordering Threshold value other than 1 is received.
+
+Largest Unacked
+: The largest packet number of a received ack-eliciting packet.
+
+Largest Acked
+: The Largest Acknowledged value sent in an ACK frame.
+
+Unreported Missing
+: Packets with packet numbers between the Largest Unacked and Largest Acked that
+  have not yet been received.
+
+An endpoint that receives an ACK_FREQUENCY frame with a non-zero Reordering
+Threshold value SHOULD send an immediate ACK when the gap
+between the smallest Unreported Missing packet and the Largest Unacked is greater
+than or equal to the Reordering Threshold value.
 
 In order to ensure timely loss detection, the Reordering Threshold provided
 in the ACK_FREQUENCY frame SHOULD NOT be larger than the re-ordering threshold
 used by the data sender for loss detection. ({{Section 18.2 of QUIC-RECOVERY}})
 recommends a default packet threshold for loss detection of 3.
 
```

Comment:
```suggestion

This will ensure that an ACK is sent at exactly the point when loss can be detected by the peer without delays. Sending this ACK will reset the ACK timer or counter as any ACK would do.
```